src/Exercicio1.ts

```diff
@@ -0,0 +1,27 @@
+
+//Alternativa A - Exemplo de uma palavra recebida via parâmetro da função
+function ContarVogais(palavra: string){
+    let qtdVogais = 0;
+    const vogais = ["a", "e", "i", "o", "u", "A", "E", "I", "O", "U"];
```

Comment:
Não é necessário um array com minúsculas e maiúsculas. Podes apenas utilizar algo como toLowerCase() pra forçar tudo para minúsculo. 
Também poderias fazer a verificação de vogais com acento.